doc/source/dev/roadmap-detailed.rst

```diff
@@ -256,9 +256,8 @@ Overall this module is in good shape. Two good global optimizers were added in
 1.2.0; large-scale optimizers is still a gap that could be filled.  Other
 things that are needed:
```

Comment:
There are some talks about refactoring in PRs like gh-8414. @andyfaff should we say something here?